src/main/java/org/polypheny/jdbc/types/PolyDocument.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2019-2024 The Polypheny Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.polypheny.jdbc.types;
+
+import static org.polypheny.prism.ProtoValue.ValueCase.STRING;
```

Comment:
Static imports are often a source of confusion and should be avoided except for a few cases like junit methods in tests.